docs/MultiNodeDeprecation.md

```diff
@@ -0,0 +1,59 @@
+## Multi-node Deprecation

```

Comment:
@mkindahl @erimatnor do you mind checking this file? This is the announcement that we have prepared with Ramon & Bryan on the product side - will be linked by other announcements as well, but it's better if it lives on the main repo